src/axom/multimat/docs/sphinx/multimat_materials.rst

```diff
@@ -0,0 +1,119 @@
+and build the CMR by populating a bool vector that is given to the MultiMat object.
+In the following example, the relation is expressed using a **Cell-Dominant**
+data layout (``CELL_DOM``). This means that the data will be arranged such that all material
+values for cell 0 are given, followed by all of the material values for cell 1, until all
+cells have provided their flags. The bool vector contains *true* if a material is present
+in a cell and *false* if the material is not present. This CMR vector is essentially
+a mask for which cell/material combinations are valid. Data can also be transposed
+into a **Material-Dominant** data layout (``MAT_DOM``) in which the materials are
+iterated first, followed by cells that use the current material.
+
+.. code-block:: cpp
+
+    constexpr int nmats = 3;
+    constexpr int ncells = 9;
+
+    // Create the MultiMat object
+    axom::multimat::MultiMat mm;
+    mm.setNumberOfMaterials(nmats);
+    mm.setNumberOfCells(ncells);
+
+    // Cell-Dominant data layout
+    int rel[9][3] = {{1,1,0},
+                     {1,1,1},
+                     {0,1,1},
+                     {1,1,1},
+                     {0,1,1},
+                     {0,0,1},
+                     {0,1,1},
+                     {0,0,1},
+                     {0,0,1}};
+    std::vector<bool> relation(ncells * nmats, false);
+    for(int c = 0; c < ncells; c++)
+    {
+      for(int m = 0; m < nmats; m++)
+      {
+        relation[c * nmats + m] = rel[c][m] > 0;
+      }
+    }
+    mm.setCellMatRel(relation, axom::multimat::DataLayout::CELL_DOM);
+
+
+#######################
+Volume Fractions
+#######################
+
+The CMR determines how materials are allocated to each cell but it does not determine
+how much of each material is present in the cell. Volume fractions determine how much of each material
+exists in each cell and is given as a percentage. If a cell contains materials A and B
+at 20% and 80%, respectively, then the volume fractions for those materials in the
+cell are: *0.2* and *0.8*. Note that the sum of volume fractions in a cell must equal 1
+to account for all of the cell. Volume fractions must be provided for every valid
+cell/material pair in the CMR and they must be specified using the same data layout
+as data in the CMR.
+
+.. figure:: figures/volume_fractions.png
+   :figwidth: 600px
+
+   Diagram showing mixed-material mesh with volume fractions shown in CELL_DOM table.
+
+Volume fractions are stored in MultiMat as a field and fields have an added concept
+of sparsity. Fields can provide data for every possible cell/material pair; this is
+called a dense field. Dense fields are easy to understand since they have values for
+every cell/material pair and they take more memory since zeroes must be provided even
+for invalid cell/material pairs. Fields can also be sparse, which eliminates the zeroes
+where a material does not exist.
```

Comment:
Suggestion: rephrase to "Dense fields are easy to understand: they have values for every cell/material pair.  Dense fields store zeros for cells where a material is not present.  Fields can also be sparse, saving memory by eliminating the zeros where a material does not exist."

(I think both "zeros" and "zeroes" are correct.  Not sure on that.)

Reply:
Changed.